### ISSUE TITLE
Weights in OPTICS

### ISSUE THREAD

**Issue by asgr** [mhahsler/dbscan#8]
This is a feature request rather than issue strictly- it seems there should be an option to have weights in OPTICS as there is for DBSCAN. Please correct me if this thinking is faulty.

**Comment by mhahsler** [comment]
You are right. I will leave it here as an enhancement. I hope I or Matt will find the time to implement it.
